### DISCUSSION TITLE
Invitation to App-Coreography

### DISCUSSION THREAD

**Discussion by uptosky524** [App-Choreography/Support#6] [General]
Hi, I hope you are doing well. I am eager to join and contribute to the Community Organization. Thank you.

**Comment by NS007-dev** [comment]
Hi @HighAmbition211 , I am so sorry for the late response, of course I will send you an invite to the org! Thank you for reaching out!

**Comment by NS007-dev** [comment]
/
###### [Our Website 🌐](https://app-choreography.github.io/) | [Repositories 📚](https://github.com/orgs/App-Choreography/repositories) | [LinkFree Profile 🔗](http://linkfree.eddiehub.org/App-Choreography)

You have received an email to join, or press this link - https://github.com/App-Choreography - , and you'll see a banner, for you to join the organisation! 🎉 

You can also show that you are part of this org, by making yourself public. This will show our profile org on your profile:
## Follow This:
1. After you join the organisation, search your name in the list of people. 🔍 
2.  Follow this below: ⬇️ 
![image](https://user-images.githubusercontent.com/70807500/130362669-7eb8ac9f-41e3-489e-b88b-1fe180e18129.png)

### When you join our org, you will be automatically be part of our community team, where you will get updates, and news about our org!

### We are so happy you have joined! Welcome to App Choreography! 🥳
